drc/DRCtech.c

```diff
@@ -4138,7 +4138,7 @@ DRCTechScale(scalen, scaled)
     int scalegcf;
 
     if (DRCCurStyle == NULL) return;
-    else if (scalen == scaled == 1) return;
+    else if (scalen == scaled) return;
```

Comment:
This needs integer truth table test ranges -1,0,1,2 the boolean table example in commit may not be enough. 

a == b == 1

Maybe:
a == 1 && b == 1 (also a == b && b == 1)
Or maybe:
a !=0 && b == 1

These are not quite the same thing.

Reply:
The scale factor is N/D (numerator / denominator), so any N/D equal to 1 means that DRCTechScale() will have no effect and does not need to be run.  So your original correction appears to be right.